doc/release-notes/rl-2103.adoc

```diff
@@ -32,6 +32,24 @@ programs.broot.verbs = [
 ];
 ----
 
+* The <<opt-programs.mpv.package>> option has been changed to allow custom
+derivations. The following configuration is now possible:
++
+[source,nix]
+----
+programs.mpv.package = (pkgs.wrapMpv (pkgs.mpv-unwrapped.override {
+  vapoursynthSupport = true;
+}) {
+  extraMakeWrapperArgs = [
+    "--prefix" "LD_LIBRARY_PATH" ":" "${pkgs.vapoursynth-mvtools}/lib/vapoursynth"
+  ];
+});
+----
+
+As a result of this change, <<opt-programs.mpv.package>> is no longer the
+resulting derivation. Use the newly introduced +opt-programs.mpv.finalPackage+
```

Comment:
```suggestion
resulting derivation. Use the newly introduced `+programs.mpv.finalPackage+`
```

Reply:
Done.

Reply:
Welp, fixed it the wrong way :cry: . Now it is really fixed (sorry about this mistake).